x/interchainstaking/keeper/receipt.go

```diff
@@ -217,6 +219,13 @@ func (k *Keeper) MintAndSendQAsset(ctx sdk.Context, sender sdk.AccAddress, sende
 	}
 
 	k.Logger(ctx).Info("Transferred qAssets to sender", "assets", qAssets, "sender", sender)
+
+	ctx.EventManager().EmitEvent(
+		sdk.NewEvent(
+			minttypes.EventTypeMint,
+			sdk.NewAttribute(sdk.AttributeKeyAmount, qAssets.AmountOf(zone.LocalDenom).String()),
+		),
+	)
```

Comment:
Should we also specify the `denom` of the coin being minted?  Here we are only emitting the amount.

Reply:
Ah, amountOf will return an math.Int, not a coin, right? 

I had assumed it was a coin in which string() would be okay. If it is an int we need the denom as @aljo242 stated

Reply:
qAssets.String() will return the amount along with the denomination in one go, do we want them separately under a new attribute or all in one go is fine?